### PR TITLE
feat(contrib): add arsenal deployment

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#19981.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#19984.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19984.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by fsamin — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20003.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20003.1 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20003.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

File: ui/proxy.conf.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "/cdsapi": {
-    "target": "http://localhost:8081",
+    "target": "http://localhost:9081",
```
Comment (richardlt) [inline ui/proxy.conf.json:3]: revert
```suggestion
    "target": "http://localhost:8081",
```

**Review by richardlt — CHANGES_REQUESTED** [review]

**Review by richardlt — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20030.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20030.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**